File: elias/Problema0271/problema0271.java
import java.util.Scanner;

public class problema0271 {
    public static void main(String[] args) {
        Scanner input = new Scanner (System.in);
        int N;
        System.out.println("Digite a quantidade de números para o vetor:");
        N = Integer.valueOf(input.nextLine());
        int[] vetor = new int[N];
        for(int i=0;i<N;i++){
            System.out.println("Digite um número para a posição "+i+" do vetor:");
            vetor[i] = Integer.valueOf(input.nextLine());
        }
        System.out.println("Números múltiplos de 2:");
        for(int i=0;i<N;i++){
            if(vetor[i] % 2 == 0){
                System.out.println(vetor[i]);
            }
        }
        System.out.println("Números múltiplos de 3:");
        for(int i=0;i<N;i++){
            if(vetor[i] % 3 == 0){
                System.out.println(vetor[i]);
            }
        }
        System.out.println("Números multiplos de 2 e de 3:");
        for(int i=0;i<N;i++){
            if(vetor[i] % 2 == 0 && vetor[i] % 3 ==0){
                System.out.println(vetor[i]);
            }
        }
    }
}

File: elias/Problema0271/problema0271.py
vetor = []
N = int(input("Digite a quantidade de números para o vetor: "))
for i in range (N):
    n = int(input("Digite um número para a posição %d do vetor:  " %i))
    vetor.append(n)
print("Números múltiplos de 2:")
for i in range (N):
    if vetor[i] % 2 == 0 :
        print(vetor[i])
print("Números múltiplos de 3:")
for i in range (N):
    if vetor[i] % 3 == 0 :
        print(vetor[i])
print("Números multiplos de 2 e de 3:")
for i in range (N):
    if vetor[i] % 2 == 0 and vetor[i] % 3 ==0 :
        print(vetor[i])
